pombola/core/tests/test_views.py

```diff
@@ -159,7 +162,7 @@ def test_position_on_person_page(self):
 
     def test_organisation_page(self):
         self.app.get('/organisation/missing-org/', status=404)
-        with self.assertNumQueries(9):
+        with self.assertNumQueries(13):
```

Comment:
I'm not sure what exactly has increased the number of queries from 9 to 13, but I'm assuming something was added to the organisation page. The 9 value was set in 2016 so I think it's safe to assume that some things have changed. The page is still fast and no duplicate or n+1-type queries were added.